src/cache/sqlite.ts

```diff
@@ -74,7 +77,17 @@ class SQLiteAdapter {
 
               return responses;
             });
-        }, Promise.resolve());
+        }, Promise.resolve())
+          .then((response) => {
+            return db.close()
```

Comment:
Yes. We were seeing the error `Too many files open`. The problem seemed to be related to open database connections because once we closed the database connections the issue went away.